pkg/utils/utils.go

```diff
@@ -62,3 +64,22 @@ func (errs combinedError) Error() string {
 	}
 	return ""
 }
+
+//LookupRequestsToNodeIDs returns the nodeIDs from the LookupRequests
+func LookupRequestsToNodeIDs(reqs *proto.LookupRequests) []string {
```

Comment:
Is there a better place where these methods could be?

Reply:
We had thought they may be useful as helpers later on, but right now they're only being used in overlay/server.go. So I could move them in there if you think that's better style.

Reply:
If they are used only in overlay/server.go, then yes... I would also consider unexporting them until actually needed to make public.

Reply:
sounds good